src/components/CopyToClipboard/CopyToClipboard.tsx

```diff
@@ -45,22 +50,33 @@ const Container = styled.span`
 `;
 
 const translateUpAndFadeIn = keyframes`
-    from { opacity: 0; transform: translateY(-14px); };
-    to { opacity: 1; transform: translateY(-28px); };
+    from { opacity: 0; transform: translateX(-50%) translateY(-14px); };
+    to { opacity: 1; transform: translateX(-50%) translateY(-28px); };
 `;
 
 const translateDownAndFadeOut = keyframes`
-    from { opacity: 1; transform: translateY(-28px); };
-    to { opacity: 0; transform: translateY(-14px); };
+    from { opacity: 1; transform: translateX(-50%) translateY(-28px); };
+    to { opacity: 0; transform: translateX(-50%) translateY(-14px); };
 `;
 
 const Toast = styled.div<{ isActive: boolean }>`
-  position: absolute;
-  background: black;
-  padding: 4px 8px;
+  position: fixed;
+  bottom: 0;
+  left: 50%;
+  z-index: 10000;
+  background: white;
+  border: 1px solid ${colors.offBlack};
+
+  border-radius: 0px;
+  padding: 8px 16px 8px 16px;
+
   width: fit-content;
   pointer-events: none;
 
+  & p {
```

Comment:
there's a toast component in Toast.tsx that we might be able to use here!
you can check for usages of `useToastActions` for examples